packages/react/src/components/UserDropdownMenu/UserDropdownMenu.tsx

```diff
@@ -194,13 +194,13 @@ const UserDropdownMenu: FC<UserDropdownMenuProps> & WithWrapperProps = (
           </ListItem>
         )}
         {menuItems?.length > 0 ? (
-          <>
+          <div>
             <Divider />
             {menuItems}
-          </>
+          </div>
```

Comment:
Why did we remove fragments and add div here?

Reply:
Using fragments inside Menu is disallowed and results in a console error.